test/index.ts

```diff
@@ -114,6 +99,18 @@ it.skip = test.skip;
 export function defaultPropTestParams<T = unknown>(): fc.Parameters<T> {
     const baseParams = {
         numRuns: Number(process.env.AZLE_PROPTEST_NUM_RUNS ?? 1),
+        reporter: (runDetails: fc.RunDetails<T>): void => {
+            const seed = runDetails.seed;
+            const path = runDetails.counterexamplePath;
+            const reproductionCommand = `AZLE_PROPTEST_SEED=${seed}${path !== null ? ` AZLE_PROPTEST_PATH="${path}"` : ''} AZLE_VERBOSE=true AZLE_TEMPLATE=true npm test`;
+            const reproductionMessage = `To reproduce this exact test case, run:\n${reproductionCommand}`;
+            console.info(reproductionMessage);
+            if (runDetails.failed) {
+                throw new Error(
```

Comment:
Are you sure this works? Are you sure tests still fail with this? I just want to double-check and be sure we didn't introduce massive amounts of false positives.

Reply:
yes I am positive, you can come see if you want

Reply:
If you look at the js docs for the reporter it tells you that the default behavior is to throw everything that fails. Also I tested it with failing tests to confirm that it still fails